celery/backends/elasticsearch.py

```diff
@@ -31,10 +32,14 @@ class ElasticsearchBackend(KeyValueStoreBackend):
     scheme = 'http'
     host = 'localhost'
     port = 9200
+    es_retry_on_timeout=False
```

Comment:
For uniformity and readability I think it is preferrable to surround `=` with spaces:

```
es_retry_on_timeout = False
```